enacts/calc.py

```diff
@@ -231,6 +260,71 @@ def sql2geom(shapes_sql, db_config):
     return df
 
 
+def synthesize_geom(bbox, level=0):
+    """ Synthesize a geometric object from a bounding box
+
+    Parameters
+    ----------
+    bbox : array
+        coordinates of bounding box of spatial domain as [W, S, E, N]
+    level : int, optional
+        0 or 1 to mimick a containing admin level (0) with 1 geometry roughly smaller
+        than `bbox` or a contained admin level (1) with 2 geometries partitioning
+        level 0. Default is 0.
+    
+    Returns
+    -------
+    df : pandas.DataFrame
+        a pd.DF with columns "label" (dtype=string),
+        "key" (string or int depending on the table),
+        and "the_geom" (shapely.Geometry)
+
+    See Also
+    --------
+    shapely.geometry.Polygon
+
+    Notes
+    -----
+    A level 0 contained into bbox is necessary to test the clipping feature since
+    `bbox` is also used to generate the fake data.
+    """
+    assert (
+        (bbox[1] + 0.25 <= bbox[3] - 0.5),
```

Comment:
This whole thing would be easier to read if you start with
```
west, south, east, north = bbox
```
and then use those names instead of `bbox[i]` in the rest of the function.